rhel/updaterset.go

```diff
@@ -136,6 +136,9 @@ func (f *Factory) UpdaterSet(ctx context.Context) (driver.UpdaterSet, error) {
 		}
 		fallthrough
 	case http.StatusNotModified:
+		// return stub updater to allow us to record that all rhel updaters are up to date
+		stubUpdater := Updater{name: "rhel-all"}
+		s.Add(&stubUpdater)
```

Comment:
It shouldn’t matter what the update period is as either the normal updater route would record successes or failures, or if the request to the vuln source shows the page is unchanged the stub updater will record all RHEL updaters as up to date as there were no changes since last time. Not sure if I understood you properly there apologies if not!

Reply:
It should work like this: 
- The initial update happens and pulls in all the RHEL vulns and each updater will record its own status as it completes or fails (not specific to RHEL updaters)
- 1 week later the RHEL updaters run again and things have changed and each updater will again record its own status 

So there are two routes to recording updater status:
- Default: each updater will record its own status as it completes or fails (https://github.com/quay/claircore/pull/558/files#diff-4010adf43a5b51ec1ae6911d0ac69fb1400e08367a3b760ee4fbc00c841fbf39R291)
- Special case: RHEL factory will not run its updaters if the vuln source is unchanged, so if we find this we update all RHEL updaters statuses at once (using the stub updater as a prompt) (https://github.com/quay/claircore/pull/558/files#diff-4010adf43a5b51ec1ae6911d0ac69fb1400e08367a3b760ee4fbc00c841fbf39R151)

Let me know if that's still unclear or I'm misunderstanding what you mean

Reply:
Gotcha, that makes sense :+1: